Update base-nl.yaml

JurgenMK opened the pull request.

Astavie commented on translations/base-nl.yaml:
```diff
@@ -30,47 +30,47 @@ steamPage:
-        Purchasing the game gives you access to the standalone version which has additional features and you'll also receive access to newly developed features.
-
-        [b]Standalone Advantages[/b]
+        [b]Losstaande voordelen[/b]
```
In the rest of the translation, "standalone" doesn't get translated, except here

Astavie commented on translations/base-nl.yaml:
```diff
@@ -30,47 +30,47 @@ steamPage:
 
-        Anybody can contribute, I'm actively involved in the community and attempt to review all suggestions and take feedback into consideration where possible.
-        Be sure to check out my trello board for the full roadmap!
+        Iedereen can toevoegen, ik ben actief in de gemeenschap, en probeer alle suggesties te bekijken en terugkoppelingen te overwegen waar mogelijk.
```
`can` should be `kan`

Astavie commented on translations/base-nl.yaml:
```diff
@@ -30,47 +30,47 @@ steamPage:
 
-        shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
-        Upon delivering the requested shapes you will progress within the game and unlock upgrades to speed up your factory.
+        shapez.io is een spel over het bouwen van fabrieken, welke steeds moeilijker wordende vormen maken en verfijnen, over een oneindig grote map.
```
I would translate `map` as `speelveld`

Astavie commented on translations/base-nl.yaml:
```diff
@@ -30,47 +30,47 @@ steamPage:
-        As the demand for shapes increases, you will have to scale up your factory to meet the demand - Don't forget about resources though, you will have to expand across the [b]infinite map[/b]!
+        Omdat de vraag naar vormen groter wordt, moet u uw fabriek steeds groter maken om aan de vraag te voldoen - vergeet de grondstoffen niet, u zal moeten groeien over de [b]oneindige map[/b]!
+        
+        Binnenkort moet u kleuren mengen en uw vormen ermee verven - Combineer rode, groene en blauwe kleurbronnen om verschillende kleuren, en verf vormen ermee om aan de vraag te voldoen.
```
"om verschillende kleuren..." this sentence isn't finished

Astavie commented on translations/base-nl.yaml:
```diff
@@ -30,47 +30,47 @@ steamPage:
+        Binnenkort moet u kleuren mengen en uw vormen ermee verven - Combineer rode, groene en blauwe kleurbronnen om verschillende kleuren, en verf vormen ermee om aan de vraag te voldoen.
 
-        Soon you will have to mix colors and paint your shapes with them - Combine red, green and blue color resources to produce different colors and paint shapes with it to satisfy the demand.
+        Dit spel heeft 18 opeenvolgende niveaus (Welke u bezig moeten houden voor al gauw een paar uur!) maar ik vroeg constant inhoud toe - er is veel gepland!
```
`welke` is a pretty outdated term. Could be changed to `die`.

Astavie commented:
I think that was everything. There's also an overabundance of commas in the entire translation, though.

JurgenMK commented:
> 
> 
> I think that was everything. There's also an overabundance of commas in the entire translation, though.

I updated the file to satisfy your comments, also removed some not really necessary commas.

Astavie commented:
Looks good 👍

tobspr commented:
Looks good! But it seems there are some conflicts?

Astavie commented:
It seems a later PR got merged before this one, leaving this one obsolete?

tobspr commented:
Closing for now
